ui/src/conferences/containers/SearchPageContainer.jsx

```diff
@@ -1,11 +1,111 @@
-import React from 'react';
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { Row, Col } from 'antd';
 import { connect } from 'react-redux';
+import { fromJS } from 'immutable';
 
-function SearchPage() {
-  return <div>Search</div>;
+import AggregationFiltersContainer from '../../common/containers/AggregationFiltersContainer';
+import PaginationContainer from '../../common/containers/PaginationContainer';
+import SortByContainer from '../../common/containers/SortByContainer';
+import ResultsContainer from '../../common/containers/ResultsContainer';
+import NumberOfResultsContainer from '../../common/containers/NumberOfResultsContainer';
+import LoadingOrChildren from '../../common/components/LoadingOrChildren';
+import ResponsiveView from '../../common/components/ResponsiveView';
+import DrawerHandle from '../../common/components/DrawerHandle';
+import DocumentHead from '../../common/components/DocumentHead';
+import { CONFERENCES_NS } from '../../reducers/search';
+import ConferenceItem from '../components/ConferenceItem';
+
+function renderConferenceItem(result) {
+  return <ConferenceItem metadata={result.get('metadata')} />;
+}
+
+const STATIC_AGGREGATIONS = fromJS({
+  start_date: {
+    meta: {
+      title: 'Start Date',
+      order: 1,
```

Comment:
I think having `1` here and starting `2` from the backend feels awkward if you read the backend code and if somebody was to add a new facet before `subject`, it's likely that they would just do `order: 1`. So maybe we can put this `0` or `-1` and start at `1` on the otherside. Or at least a comment there?

Reply:
or another idea would be to put the staticAggregation always first and not add order for them at all, wdyt? I did it like that initially because I thought maybe in the future, the staticAggregation would even be between 2 aggregations, but you are right, it is confusing for the backend and there is no need to have it like this yet.

Reply:
I was thinking to make them always first, but you know how it goes.. we will probably need some of them last or sth like that in the future.

Reply:
I still say let's go always first for now. And if needed, we will change it later. Adding ```-1``` and things like that isn't too nice